api/models.cs

```diff
@@ -19,4 +19,26 @@ public class ResolveRequest
 {
     [FromRoute(Name = "shortUrl")] // wouldnt work without this
     public string ShortURL { get; set; }
+}
+
+
+public class RedisActions
```

Comment:
It is not model, so it makes sense to move it in separate folder. I usually name it Services

Reply:
done

Reply:
done